src/fromager/commands/build.py

```diff
@@ -125,13 +133,20 @@ def build(
     "cache_wheel_server_url",
     help="url to a wheel server from where fromager can check if it had already built the wheel",
 )
+@click.option(
+    "--local-version",
+    default="",
+    type=str,
```

Comment:
local version are [restricted](https://packaging.python.org/en/latest/specifications/version-specifiers/#local-version-identifiers) to alphanumeric ASCII and dots. Should we add a [validator](https://click.palletsprojects.com/en/stable/options/#callbacks-for-validation)?

Reply:
Good idea. I was going to rely on the Version class parsing, but it's better to do it earlier.